### PR TITLE
[MO IR Reader] Update *Sequence backend_attrs

### PR DESCRIPTION
**Pull request by achetver**
Root cause analysis: Now we have inconsistency between MO `backend_attrs` attribute of `LSTMSequence` operation and [specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/LSTMSequence_1.md). It leads to errors with missed data while graph reading and saving by MO IR Reader.

Solution: Align `backend_attrs` attribute values with specification for correct saving all operation attributes for all related operations. Also added extender for `GRUSequence` operation.

Ticket: 76816, 76800, 77582

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names
* [x]  Transformation preserves tensor names

Validation:
* [x]  Unit tests: N/A - No new implemented functions;
* [x]  Framework operation tests: N/A - No new enabled operations;
* [x]  Transformation tests: N/A - No new implemented transformations;
* [x]  Model Optimizer IR Reader check: In progress.

Documentation:
* [x]  Supported frameworks operations list: N/A - No new supported operations;
* [x]  Supported **public** models list: N/A - Nothing to add;
* [x]  User guide update: N/A - Nothing to update.

### CODE REVIEW

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py [hunk 1]
```diff
@@ -60,6 +60,10 @@ def supported_attrs(self):
     def backend_attrs(self):
         return [
             'hidden_size',
+            ('activations', lambda node: ','.join(node['activations'])),
+            ('activations_alpha', lambda node: ','.join(map(str, node['activations_alpha']))),
+            ('activations_beta', lambda node: ','.join(map(str, node['activations_beta']))),
+            'direction',
         ]
```
Comment (vgavrilo) [inline tools/mo/openvino/tools/mo/ops/lstm_sequence.py:67]: Should the attribute `clip` be added here?

**Review by vgavrilo — CHANGES_REQUESTED** [review]

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py (hunk 1 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/ops/lstm_sequence.py:68]: Yes it should, added.

**Review by vgavrilo — APPROVED** [review]

**Review by pavel-esir — APPROVED** [review]

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py [hunk 2]
```diff
@@ -60,6 +60,11 @@ def supported_attrs(self):
     def backend_attrs(self):
         return [
             'hidden_size',
+            ('activations', lambda node: ','.join(node['activations'])),
+            ('activations_alpha', lambda node: ','.join(map(str, node['activations_alpha']))),
+            ('activations_beta', lambda node: ','.join(map(str, node['activations_beta']))),
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/ops/lstm_sequence.py:65]: please check these attributes as well

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py [hunk 3]
```diff
@@ -60,6 +60,11 @@ def supported_attrs(self):
     def backend_attrs(self):
         return [
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/ops/lstm_sequence.py:61]: Also, do we need extender for this operation? 
Also, please check other RNNSequence operations?

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py [hunk 4]
```diff
@@ -60,6 +60,11 @@ def supported_attrs(self):
...
             'hidden_size',
+            ('activations', lambda node: ','.join(node['activations'])),
```
[the remark this answers is not in the document]
Comment (achetver) [reply tools/mo/openvino/tools/mo/ops/lstm_sequence.py]: Updated for all related operations.

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py (hunk 2 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/ops/lstm_sequence.py]: Done, updated.

File: tools/mo/openvino/tools/mo/ops/lstm_sequence.py (hunk 3 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/ops/lstm_sequence.py:61]: Added handling for GRUSequence operation.

**Comment by rkazants** [comment]
@achetver, please chech e2e test TF_GNMT_api_2_False_batch_1_device_CPU_fusing_True_precision_FP32 that is failling during IR emiting

**Comment by achetver** [comment]
> @achetver, please check e2e test TF_GNMT_XXX that is failling during IR emiting

Checked and fixed, waiting for new e2e results

**Review by rkazants — CHANGES_REQUESTED** [review]
Please take a look at DeepSpeech models in the precommit. IR generation failed for them

**Comment by achetver** [comment]
> Please take a look at DeepSpeech models in the precommit. IR generation failed for them

These failures wasn't related to my changes and caused by infrastructure problems. After rebuild CI status is green.

**Pull request merged**